spec/features/sessions/user_sign_in_spec.rb

```diff
@@ -56,6 +67,9 @@
       it 'performs a successful login' do
         page.click_link 'Sign in with HPI OpenID Connect'
         expect(page).to have_text('Logout')
+        #expect(page).to have_text 'Signed in successfully.'
+        expect(page).to have_text 'Successfully authenticated'
```

Comment:
pls move this to a separate test which tests if there is a flash message

Reply:
but then i would have to write the exact same test?

Reply:
I see will separate in the same file